horovod/ray/elastic_v2.py

```diff
@@ -162,6 +162,8 @@ class ElasticParams(BaseParams):
         reset_limit (int): Maximum number of times that the training
             job can scale up or down the number of workers after
             which the job is terminated.
+        cooldown_range(int range): Range of seconds(min, max) a failing
+            host will remain in blacklist.
```

Comment:
Can you given an example, I think `int range` is not very common to know that two integers are expected here.

Reply:
Done